ibexa/commerce/4.0.x-dev/encore/ez.webpack.config.manager.js

```diff
@@ -1,33 +1,44 @@
 const fs = require('fs');
-const findItems = (eZConfig, entryName) => {
-    const items = eZConfig.entry[entryName];
+const ibexifyEntryName = (entryName) => {
```

Comment:
Could you explain in a comment, why we need to ibexify entry name?

Reply:
Ok, so basically, we changed all our entry names from `ezplatform` to `ibexa`, but clients can have parts of a code in which they extend our entries and still relay on old entries' names. We want to assure that in version 4.0 these old entry names in such extensions still work.